### PR TITLE
Add hardware configurations and aliases to 5.4 tpch

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/issues/9137, https://github.com/pingcap/docs-cn/pull/9140

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/v5.4-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TiDB TPC-H 性能对比测试报告 - v5.4 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
+aliases: ['/zh/tidb/stable/v5.4-performance-benchmarking-with-tpch/']
```
Comment (TomShawn) [inline benchmark/v5.4-performance-benchmarking-with-tpch.md:3]:
```suggestion
aliases: ['/zh/tidb/stable/v5.4-performance-benchmarking-with-tpch/','/zh/tidb/v5.4/v5.4-performance-benchmarking-with-tpch/']
```

**Review by TomShawn — APPROVED** [review]

File: benchmark/v5.4-performance-benchmarking-with-tpch.md (hunk 1 above)
Comment (TomShawn) [inline benchmark/v5.4-performance-benchmarking-with-tpch.md:3]: 如果 release-6.0 分支没删文件，这里就不用加 aliases

Comment (shichun-0415) [reply benchmark/v5.4-performance-benchmarking-with-tpch.md:3]: 多谢 Tom 提醒，我感觉还是删掉比较合适，6.0 分支和 master 分支都放 6.0 的测试报告。

File: benchmark/v5.4-performance-benchmarking-with-tpch.md [hunk 2]
```diff
@@ -18,7 +19,13 @@ TiDB v5.0 中引入的 [TiFlash](/tiflash/tiflash-overview.md) 组件的 MPP 模
...
 ### 硬件配置
 
-+ 节点数量：3
+|  实例类型  |  实例数  |
+|:----------|:----------|
+| PD        |  |
+| TiDB     |  |
+| TiKV     |  |
+| TiFlash   |  |
+
```
Comment (moyun) [inline benchmark/v5.4-performance-benchmarking-with-tpch.md:28]: PD = 1
TiDB = 1
TiKV = 3
TiFlash = 3

**Pull request merged**